### PR TITLE
Rebuild 2.1

### PR DESCRIPTION
**Pull request by SC426**
apscheduler 3.9.0 --> tzlocal 2.1

Needed to rebuild for s390x

### CODE REVIEW

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+channels:
+  melody_org: flask-update
```
Comment (katietz) [inline abs.yaml:2]: please remove this channels stuff from commit

File: recipe/meta.yaml [hunk 2]
```diff
@@ -9,13 +9,14 @@ source:
...
 build:
-  number: 0
+  number: 1
   script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   host:
     - python
     - pip
+    - wheel
```
Comment (katietz) [inline recipe/meta.yaml:19]: missing setuptools

File: recipe/meta.yaml [hunk 3]
```diff
@@ -24,11 +25,23 @@ requirements:
...
+    for pytz compatibility) with the local timezone information, 
+    under Unix and Windows.
+  doc_url: https://github.com/regebro/tzlocal/blob/{{ version }}/CHANGES.txt
+  doc_source_url: https://github.com/regebro/tzlocal/tree/{{ version }}
   home: https://github.com/regebro/tzlocal
-  license: MIT
+  license: MIT 
+  license_family: MIT
   license_file: LICENSE.txt
+  license_url: https://github.com/regebro/tzlocal/blob/{{ version }}/LICENSE.txt
```
Comment (katietz) [inline recipe/meta.yaml:44]: no need for this line, we have already lincense_file specified.  That URL is superflous

File: recipe/meta.yaml [hunk 4]
```diff
@@ -24,11 +25,23 @@ requirements:
...
+  commands:
+    - pip check
 
 about:
+  description: |
+    This Python module returns a tzinfo object (with a pytz_deprecation_shim, 
+    for pytz compatibility) with the local timezone information, 
+    under Unix and Windows.
+  doc_url: https://github.com/regebro/tzlocal/blob/{{ version }}/CHANGES.txt
+  doc_source_url: https://github.com/regebro/tzlocal/tree/{{ version }}
```
Comment (katietz) [inline recipe/meta.yaml:39]: we have doc_url, no need for doc_source_url.  In general "doc_source_url" is pretty much superflous if there is already a doc_url tag

**Review by katietz — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (SC426) [reply recipe/meta.yaml:20]: This has been added

File: recipe/meta.yaml (hunk 3 above)
Comment (SC426) [reply recipe/meta.yaml]: This has been removed

File: recipe/meta.yaml (hunk 4 above)
Comment (SC426) [reply recipe/meta.yaml:39]: This has been removed

File: abs.yaml (hunk 1 above)
Comment (SC426) [reply abs.yaml]: The abs.yaml has been removed

**Review by varlackc — APPROVED** [review]
LGTM

**Comment by SC426** [comment]
> LGTM

Thank you!

**Review by ELundby45 — APPROVED** [review]

**Pull request merged**